setup.cfg

```diff
@@ -37,7 +37,7 @@ install_requires =
     pyxdg >=0.26
     requests >=2.24
     tomli>=1.2.3; python_version<'3.11'
-    urllib3 >= 1.26.5, < 2.0.0
+    urllib3
```

Comment:
Renovate wants this to be `    urllib3 >=2.0.3, <2.1.0`.  Matching `requirements.txt`.